changelog/2.075.1.dd

```diff
@@ -28,7 +28,7 @@ $(LI $(BUGZILLA 17192): ParameterDefaults fails when parameter is called "i")
 )
 $(BUGSTITLE Phobos bugs,
 
-$(LI $(BUGZILLA 17667): regex$(LPAREN)[r".", r"[\$(LPAREN)\{[\]\}\$(RPAREN)]"]$(RPAREN);)
+$(LI $(BUGZILLA 17667): regex$(LPAREN)[r".", r"[\\$(LPAREN)\\{[\\]\\}\\$(RPAREN)]"]$(RPAREN);)
```

Comment:
Better wrap the whole thing in backticks, which will also remove the need for macros like `$(LPAREN)`.